content/doc/book/managing/ui-themes.adoc

```diff
@@ -0,0 +1,99 @@
+
+## Using themes
+
+// TODO: Add configuration examples and screenshots one documentation is added to plugins
+
+To configure a theme, you will need to install a theme management plugin and then to configure it.
+Both plugins can be managed in the Jenkins Web UI or by the plugin:configuration-as-code[Configuration-as-Code Plugin].
+See the plugin documentation for the detailed usage guidelines.
+
+Example for the link:https://tobix.github.io/jenkins-neo2-theme/[Jenkins Neo2] theme:
+
+image::managing/simple-theme-plugin-neo2.png["Jenkins Neo2 Theme", role=center]
+
+== Themes support policy
+
+WARNING: Jenkins themes are provided “as is”, without warranty of any kind, implicit or explicit.
+The Jenkins core, plugins and other component updates may break theme compatibility without notice.
+
+At the moment Jenkins project does not provide specification for layouts/CSS,
+and we cannot guarantee backward or forward compatibility.
+We try to reflect major changes in changelogs
+(e.g. see the ‘developer’ changes in the link:/changelog/[Jenkins changelog]),
+but minor changes may not be included there.
+
+=== Why?
+
+There is an ongoing effort focused on improving Jenkins look-and-feel, accessibility, and user experience.
+This area is mission-critical to the project.
+Nowadays Jenkins user interface is widely considered dated, and we want to change this perception.
+There are multiple initiatives in the link:/project/roadmap/[Jenkins Roadmap] being coordinated by the link:/sigs/ux/[Jenkins User Experience SIG].
+
+Major UI changes imply incompatible changes in layouts and the CSS structure which is critical for theme plugins.
+Historically Jenkins had no explicit support policy for themes,
+and we do not want to provide compatibility requirements which would create obstacles for reworking the main Jenkins interface.
+Later, once the Jenkins UI rework reaches its destination and the UI becomes more stable, we could consider creating specifications for theme extensibility so that we could make themes more stable and maintain compatibility.
+
+== Reporting and fixing issues
+
+Users are welcome to report discovered compatibility issues to theme maintainers,
+and to submit patches there.
+It is possible to just fix a theme locally,
+but other theme users would appreciate compatibility fixes in upstream repostiories .
+
+We will generally reject bug reports to the Jenkins core/plugins involving broken UI when a custom theme.
+We will consider pull requests which restore compatibility and do not block further Web UI evolvement.
+
+NOTE: If a theme outside the link:https://github.com/jenkinsci[jenkinsci] GitHub organization is no longer maintained,
+it is fine to fork it and to create a new version.
+For themes hosted within the `jenkinsci` organization,
+we have an link:/doc/developer/plugin-governance/adopt-a-plugin/[adoption process] which also applies to themes.
```

Comment:
is this any themes currently hosted in the org?

Reply:
Not yet, but probably we will host the Dark theme created during the hackfest. I would be also fine with moving Neo2 theme if @Tobix is interested

Reply:
Sure, we can do that. Just have to think about how not to break the currently deployed GitHub pages links 🤔

Reply:
Good question. Not sure whether GitHub implements automatic redirects in such case. We could use another repo for testing